test/test_output_readers.jl

```diff
@@ -390,7 +390,7 @@ end
         @testset "FieldDataset{$Backend} parallel reading" begin
             @info "  Testing FieldDataset{$Backend} parallel reading..."
 
-            reader_kw = Dict(:parallel_read => true)
+            reader_kw = (parallel_read = true,)
```

Comment:
```suggestion
            reader_kw = (; parallel_read = true)
```

preferred syntax (less prone to typos)